packages/foundry/README.md

```diff
@@ -59,11 +59,13 @@ Use the [static-module](https://github.com/workleap/wl-foundry-cli/tree/main/tem
 
 Use the [web-application](https://github.com/workleap/wl-foundry-cli/tree/main/templates/web-application) template as a base to customize with these options:
 
-| option                      | description                  | required |
-|-----------------------------|------------------------------|----------|
-| -o, --out-dir <string>      | where to create the template | ✔        |
-| -n, --package-name <string> | package name                 | ✔        |
-| -h, --help                  | display help for command     |          |
+| option                        | description                                         | required |
```

Comment:
Maybe we should remove "short" alias, I feel they are more confusing than helpful, what do you think?

Reply:
It could be a good idea to remove them while there are no command overlaps and no user.

Even in my tests, I don't use the short version... and I think having less alias for a parameter is a good idea.